src/home.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const Home = (props) => (
```

Comment:
This component is a form, so having `Form` in the name would be better, like `FlashCardForm`.